### DISCUSSION TITLE
How to use fingerprintjs-android in Java

### DISCUSSION THREAD

**Discussion by touhidrasel** [fingerprintjs/fingerprintjs-android#49] [Q&A]
I can not import the Fingerprinter class from fingerprintjs-android library. There is documentation to add a dependency for java but after I have added the library I can not import any class from it. I am using Android studio and I am writing java code. 
Android Studio does not recognize "Fingerprinter" as an object.
Am I missing something? Is there any special import statement I need to use? Please help.

**Comment by Alexey-Verkhovsky** [comment]
Hello! 

Make sure you declared the right repository (jitpack), you can do it either in the projects `build.gradle` file (if your AGP version below 7.0) or in the `settings.gradle` file (if your AGP version is 7.0 or higher):

```groovy
allprojects {	
  repositories {
  ...
  maven { url 'https://jitpack.io' }	
}}
```

And then don't forget to declare the dependencies. Our library is written on Kotlin, so import `kotlin-stdlib` as well. Write these lines in the `build.gradle` file of the module where you plan to use the library.
```groovy
dependencies {
  // Add this line!!!
  implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"

  implementation "com.github.fingerprintjs:fingerprint-android:1.2"
}

```
